config/nozk-broker.properties

```diff
@@ -68,7 +71,7 @@ socket.request.max.bytes=104857600
 ############################# Log Basics #############################
 
 # A comma separated list of directories under which to store log files
-log.dirs=/tmp/raft-broker-logs
+log.dirs=/tmp/nozk-broker-logs
```

Comment:
Instead of "nozk" in these configurations, how about "self-managed"?

Reply:
I like nozk better since it makes it really clear.  I wouldn't want people to be using this by accident.

I think once we're out of early-access / preview / etc we'll just rename this to broker.conf, etc. and that will be cleaner.

Reply:
Ok, I don't feel strongly about it.